spec.bs

```diff
@@ -803,23 +804,24 @@ An interest group is a [=struct=] with the following items:
 :: Null or an [=ordered map=] whose [=map/keys=] are [=strings=] and whose [=map/values=] are
   {{double}}. Overrides the {{AuctionAdConfig}}'s corresponding priority signals.
 : <dfn>execution mode</dfn>
-:: A [=string=], defaulting to "compatibility".
+:: A [=string=], defaulting to "compatibility". Acceptable values are "compatibility" and "group-by-origin".

```

Comment:
Usual practice is to wrap raw strings in `<code>` blocks like I suggest above

Reply:
Done